scripts/tests/scripts_regression_tests.py

```diff
@@ -2580,7 +2584,7 @@ class G_TestBuildSystem(TestCreateTestCommon):
     ###########################################################################
     def test_clean_rebuild(self):
     ###########################################################################
-        casedir = self._create_test(["--no-run", "SMS.f19_g17_rx1.A"], test_id=self._baseline_name)
+        casedir = self._create_test(["--no-run", "SMS.f19_g16_rx1.A"], test_id=self._baseline_name)
```

Comment:
I think that this needs to be g17 for cesm, we have dropped the g16 grid.

Reply:
This grid is used in cime_developer test suite, so it should work for all models.